src/plugins/streamerModeOnStream/index.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2024 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { FluxDispatcher, UserStore } from "@webpack/common";
+
+function startStreamerMode(data) {
+    const userID = UserStore.getCurrentUser().id;
+    const streamKey = data.streamKey.split(":")[3];
+    if (streamKey !== userID) return;
+
+    FluxDispatcher.dispatch({
+        type: "STREAMER_MODE_UPDATE",
+        key: "enabled",
+        value: true
+    });
+}
+
+function stopStreamerMode(data) {
```

Comment:
please type your arguments
```suggestion
function stopStreamerMode(data: { streamKey: string }) {
```